dashboard/test/models/script_test.rb

```diff
@@ -58,7 +58,7 @@ def populate_cache_and_disconnect_db
     assert script.login_required?
     assert_equal 'Level 1', script.levels[0].name
 
-    assert_equal false, Script.find(2).login_required?
+    assert_equal false, Script.find_by(name: 'frozen').login_required?
```

Comment:
Changed this to frozen so I could remove Hour of Code from the generate fixtures list. We use frozen in other places so this just saves us loading another script.